src/app/component/AddToCart/AddToCart.container.js

```diff
@@ -123,27 +119,34 @@ export class AddToCartContainer extends PureComponent {
         }).then(() => this._afterAdded());
     }
 
+    removeProductFromWishlist() {
```

Comment:
Maybe this should happen in dispatcher? So every time you add to cart, we try removing the product from wish-list? Or there is to small amount of data to remove from wish-list?

Reply:
There is not much being removed from wishlist, if I correctly understood statement.. However, it brings some challenges like accessing wishlist store from the dispatcher / requiring wishlistItems otherwise. And we are using `addToCart` method, when changing qty in cart, so In my opinion, it would be better to leave it here. I've been thinking on how events (or observerers) approach could be used here, but did not came with "easy&cheap" solution at a moment.